src/graphics/GraphvizWiringDiagrams.jl

```diff
@@ -416,6 +423,7 @@ function to_graphviz_property_graph(d::UndirectedWiringDiagram;
   # Create nodes for boxes.
   box_vs = add_vertices!(graph, nboxes(d))
   set_vprop!(graph, box_vs, :id, [ "box$b" for b in boxes(d) ])
+  set_vprop!(graph, box_vs, :fillcolor, [ "green" for b in boxes(d) ])
```

Comment:
Sorry, one more thing. Was this change intended? I feel like the UWD boxes probably shouldn't have a color by default, but if they are colored, that color should at least be configurable.

Reply:
definitely unintended, thanks!